.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+before_install:
+  - docker-compose -v
+install:
+  - docker-compose build app
```

Comment:
docker-compose tady neni na nic potreba, proc to neni podle templaty? https://github.com/keboola/component-generator/blob/master/templates/php-component/.travis.yml#L9-L12

Reply:
revertnuto